### PR TITLE
Updating PyBEL processor for PyBEL 12

### PR DESCRIPTION
**Pull request by cthoyt**
Closes pybel/pybel#331

### CODE REVIEW

**Comment by cthoyt** [comment]
Sorry in advance, I had to remove all the parentheses around the tuples to make PyCharm stop complaining :p

File: indra/statements.py [hunk 1]
```diff
@@ -1074,7 +1074,7 @@ class Statement(object):
 
     Parameters
     ----------
-    evidence : list of :py:class:`Evidence`
+    evidence : None or :py:class:`Evidence` or list of :py:class:`Evidence`
```
Comment (johnbachman) [inline indra/statements.py]: What would cause evidence to be None? If a statement is synthetic it should just be an empty list.

Comment (johnbachman) [reply indra/statements.py]: Never mind!

**Comment by johnbachman** [comment]
One question--because the hash/key change is specific to pybel/develop, merging this will make INDRA incompatible with pip-installed pybel. Will this be merged into master and released in the near future?

**Comment by cthoyt** [comment]
Changing the way the hashes are stored required changing a lot of functions that handled these operations. Luckily, I put a lot of time into unit testing each so now it's pretty easy to find any issues that pop up. I'd like to release these changes to PyPI soon (possibly with the networkx 2.1 update)

However, I'm unclear on how INDRA does dependencies. Do you have a version of PyBEL pinned, or is it always pointing to the latest on GitHub?

**Pull request merged**